pkg/handlers/webhookrhobsreceiver.go

```diff
@@ -95,17 +97,71 @@ func (h *WebhookRHOBSReceiverHandler) processAMReceiver(d AMReceiverData, ctx co
 			continue
 		}
 
-		err = h.processAlert(alert, mfn)
-		if err != nil {
-			log.WithError(err).Error("a firing alert could not be successfully processed")
+		// Handle firing alerts
+		if alert.Status == string(model.AlertFiring) {
+			err = h.processFiringAlert(alert, mfn)
+			if err != nil {
+				log.WithError(err).Error("a firing alert could not be successfully processed")
+			}
+			continue
+		}
+
+		// Handle resolving alerts
+		if alert.Status == string(model.AlertResolved) {
+			err = h.processResolvedAlert(alert, mfn)
+			if err != nil {
+				log.WithError(err).Error("a resolving alert could not be successfully processed")
+			}
+			continue
 		}
 	}
+
 	return &AMReceiverResponse{Error: nil, Status: "ok", Code: http.StatusOK}
 }
 
-// processAlert handles the pre-check verification and sending of a notification for a particular alert
+// processResolvedAlert handles resolve notifications for a particular alert
+// currently only handles removing limited support
+func (h *WebhookRHOBSReceiverHandler) processResolvedAlert(alert template.Alert, mfn oav1alpha1.ManagedFleetNotification) error {
```

Comment:
A question here: The alert for the LS is a resolved one? Or the alert for LS can be a firing one? If the alert has been resolved why we still need to put the cluster to limited support? So maybe the name `processLimitSupport` is better?

Reply:
The alert for LS can be both firing and resolving:
- if the alert is firing and contains `limited_support = true`, it will be put in limited support
- if the alert is resolving and contains `limited_support = true`, it will be removed from limited support.

I kept the naming of this method neutral to keep it extensible for the future and be in line with the way the standard webhook was implemented. 
This method could handle resolving alerts in a different ways (e.g. `send resolved servicelog` for the non-RHOBS webhooks) in the future. But you are correct, for now this function only handles resolving alerts that contain the `limited_support` label, for the rest it would return `nil`.